newrelic.js

```diff
@@ -0,0 +1,69 @@
+'use strict'
+/**
+ * New Relic agent configuration.
+ *
+ * See lib/config/default.js in the agent distribution for a more complete
+ * description of configuration variables and their potential values.
+ */
+exports.config = {
+  /**
+   * Array of application names.
+   */
+  app_name: ['SDCProxy'],
+  /**
+   * Your New Relic license key.
+   */
+  license_key: '739d9ab4aef302b2a8c010fbaa519cea8e6fNRAL',
```

Comment:
@WDHudson invalidate your key with new relic! untrack this file and gitignore it